resources/keywords.resource

```diff
@@ -935,3 +937,17 @@ Set Up EnvelopedData Certs For Tests
     Issue POS RSA KTRI Cert
     Issue NEG ECC KARI Cert
     Issue POS ECC KARI Cert
+
+Wait Until Server Revoked Cert
+    [Documentation]    Wait time until the server revoked a certificate so that testing can be begin.
+    Sleep    ${REVOKED_WAIT_TIME}
+
+Wait Until Server Updated Cert
+    [Documentation]    Wait time until the server updated a certificate so that testing can be begin.
+    ...                Needed in some cases, if a updated certificate is treated as a revoked certificate,
+    ...                so that no new certificates can be issued.
+    Sleep    ${REVOKED_WAIT_TIME}
+
+Wait Until Server Revived Cert
+    [Documentation]    Wait time until the server revived a certificate so that testing can be begin.
+    Sleep    ${REVOKED_WAIT_TIME}
```

Comment:
Sleeping for hardcoded intervals of time is probably not a good thing.

Use [Wait Until Keyword Succeeds](https://robotframework.org/robotframework/latest/libraries/BuiltIn.html#Wait%20Until%20Keyword%20Succeeds) with a timeout instead.